### PR TITLE
LL 6649 - Rebrand Icons

### PR DESCRIPTION
**Pull request by juan-cortes**
Brings a component generator from raw svg icons such as the one we have on live-common currently. It also exposes a `Icon` component that allows us to specify a name and a weight for an icon. Typing can be improved on it but it's a good first version. For testing, simply make sure that the icons are displayed and the props affect the rendering.

### Type

Rebranding - UI

### Context

https://ledgerhq.atlassian.net/browse/LL-6649

### CODE REVIEW

File: ui-lib/scripts/buildReactIcons.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+const fs = require("fs");
+const glob = require("glob");
+const camelcase = require("camelcase");
+const path = require("path");
+const svgr = require("@svgr/core").default;
+
+const rootDir = path.join(__dirname, "../src/assets/icons");
+const reactDir = `${rootDir}`;
+// const reactNativeDir = `${rootDir}/reactNative`;
```
Comment (jgiamporcaro-ledger) [inline ui-lib/scripts/buildReactIcons.js:9]: Can we just remove comments from this file, please?

**Review by jgiamporcaro-ledger — APPROVED** [review]

File: ui-lib/scripts/buildReactIcons.js (hunk 1 above)
Comment (juan-cortes) [reply ui-lib/scripts/buildReactIcons.js]: Sure, removed all traces of rn code

**Review by jgiamporcaro-ledger — APPROVED** [review]

**Comment by ggilchrist-ledger** [comment]
LGTM

**Comment by juan-cortes** [comment]
pending a ✔️ from live-core team to merge (cc @valpinkman @MortalKastor @machard @gre @JunichiSugiura)

**Review by valpinkman — APPROVED** [review]

**Pull request merged**